torchvision/prototype/transforms/_utils.py

```diff
@@ -11,11 +11,12 @@
 
 def query_bounding_box(sample: Any) -> features.BoundingBox:
     flat_sample, _ = tree_flatten(sample)
-    for i in flat_sample:
-        if isinstance(i, features.BoundingBox):
-            return i
-
-    raise TypeError("No bounding box was found in the sample")
+    bounding_boxes = {item for item in flat_sample if isinstance(item, features.BoundingBox)}
+    if not bounding_boxes:
+        raise TypeError("No bounding box was found in the sample")
+    elif len(bounding_boxes) > 2:
+        raise TypeError("Found multiple bounding boxes in the sample")
```

Comment:
Maybe, we can add some hints on how to solve the issue in both error messages ?
Especially for "Found multiple bounding boxes in the sample", where the error means that user passes batched sample instead of a single datapoint